File: java/Adapter/complete/Adapter.java
public class Adapter{
  public Adaptee adaptee;

  public Adapter(String name, int age, String eye_color){
    adaptee = new Adaptee(name, age, eye_color);
  }

  public String get_name(){
    return adaptee.name;
  }

  public void set_name(String name){
    adaptee.name = name;
  }

  public int get_age(){
    return adaptee.age;
  }

  public void set_age(int age){
    adaptee.age = age;
  }

  public String get_eye_color(){
    return adaptee.eye_color;
  }

  public void set_eye_color(String eye_color){
    adaptee.eye_color = eye_color;
  }
}

File: python/Adapter/complete/Adapter.py
from Adaptee import Adaptee

class Adapter(object):

    def __init__(self, name, age, eye_color):
        self.adaptee = Adaptee(name, age, eye_color)

    def get_name(self):
        return self.adaptee.name

    def set_name(self, name):
        self.adaptee.name = name

    def get_age(self):
        return self.adaptee.age

    def set_age(self, age):
        self.adaptee.age = age

    def get_eye_color(self):
        return self.adaptee.eye_color

    def set_eye_color(self, eye_color):
        self.adaptee.eye_color = eye_color
